packages/client/hmi-client/src/components/model/tera-model.vue

```diff
@@ -124,6 +125,10 @@ const isNaming = computed(() => isEmpty(props.assetId) || isRenaming.value);
 const hasChanged = computed(() => !isEqual(model.value, temporaryModel.value));
 const hasEditPermission = useProjects().hasEditPermission();
 
+const route = useRoute();
+
+const isWorkflow = computed(() => route.params.pageType === 'workflow');
```

Comment:
Don't think this has to be computed. Maybe passing an `isInWorkflow` prop would be better